x-pack/plugins/cases/public/components/user_actions/use_user_actions_handler.tsx

```diff
@@ -122,7 +129,7 @@ export const useUserActionsHandler = (): UseUserActionsHandler => {
   );
 
   useEffect(() => {
-    if (draftComment?.commentId) {
+    if (draftComment?.commentId && draftComment?.commentId !== 'description') {
```

Comment:
Is this one specifically for when we're dealing with a lens for a comment? Should we check for `comment` or whatever it is in that case instead?

Reply:
Here the `draftComment` could have either `newComment Id `or `any of the existing comment id`, which will be difficult to get in useEffect. 
However we specifically don't  want to run this useEffect for `description`. That's why it is added.